App.tsx

```diff
@@ -1,22 +1,30 @@
 import React, { FC } from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
+import { Provider } from 'react-redux';
```

Comment:
```suggestion
import { Provider as ReduxProvider } from 'react-redux';
```

redux provider 라는걸 직관적으로 보여주면 좋을거같아요!

Reply:
아 넵넵 동의합니다